e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -91,15 +93,28 @@ describeEE("scenarios > embedding-sdk > interactive-question", () => {
       expect(response?.statusCode).to.equal(202);
     });
 
-    tableInteractive().findByText("Max of Quantity").should("be.visible");
+    const columnNames = ["Product ID", "Max of Quantity"];
 
-    tableHeaderClick("Max of Quantity");
+    columnNames.forEach((columnName, index) => {
+      tableInteractive().findByText(columnName).should("be.visible");
 
-    popover()
-      .findByTestId("click-actions-sort-control-formatting-hide")
-      .click();
+      tableHeaderClick(columnName);
 
-    tableInteractive().findByText("Max of Quantity").should("not.exist");
+      popover()
+        .findByTestId("click-actions-sort-control-formatting-hide")
+        .click();
+
+      if (index < columnNames.length - 1) {
```

Comment:
nit: looks like this only evaluates to `false` on the last column. considering that we only have two columns here, we could probably hard-code the column to make it easier to read.

Reply:
Do you mean something like `if (columnName === "Max of Quantity") {...}`?

Reply:
@sanex3339 yes, seems a little bit easier to read that way (as we only have two columns)